fixieai/agents-ts/package.json

```diff
@@ -15,8 +15,11 @@
     "format": "dprint fmt",
     "format:check": "dprint check",
     "postinstall": "npm run build",
-    "build": "tsc -p .",
+    "build": "tsc -p . && chmod a+x dist/serve-bin.js",
     "start:dev": "npx nodemon",
+    "bin:dev": "npx ts-node src/serve-bin.ts",
+    "unit": "npx jest --runInBand",
```

Comment:
I had to do `runInBand` because otherwise the tests will try to launch servers on the same port.